codec-http/src/main/java/io/netty/handler/codec/http/HttpObjectDecoder.java

```diff
@@ -176,9 +181,20 @@ protected HttpObjectDecoder(
              DEFAULT_INITIAL_BUFFER_SIZE);
     }
 
+    /**
+     * Creates a new instance with the specified parameters.
+     */
     protected HttpObjectDecoder(
             int maxInitialLineLength, int maxHeaderSize, int maxChunkSize,
             boolean chunkedSupported, boolean validateHeaders, int initialBufferSize) {
+        this(maxInitialLineLength, maxHeaderSize, maxChunkSize, chunkedSupported, validateHeaders, initialBufferSize,
+             DEFAULT_ALLOW_DUPLICATE_CONTENT_LENGTHS);
+    }
+
+    protected HttpObjectDecoder(
+            int maxInitialLineLength, int maxHeaderSize, int maxChunkSize,
+            boolean chunkedSupported, boolean validateHeaders, int initialBufferSize,
+            boolean allowDuplicateContentLengths) {
```

Comment:
Would be nice to add this param to the class javadoc, after description of `maxInitialLineLength`, `maxHeaderSize`, and `maxChunkSize`. 
Other missing params (`chunkedSupported`, `validateHeaders`, and `initialBufferSize`) can be documented in a follow-up PR.

Reply:
I agree that it should be documented, I just couldn't find a great place to include it (since nothing is currently documented). I will take the liberty to add it to the class Javadoc.